src/main/java/org/mskcc/smile/commons/impl/JsonComparatorImpl.java

```diff
@@ -205,7 +206,14 @@ private String findPrimaryIdFromJsonNode(JsonNode sampleNode) {
      */
     private String standardizeAndFilterRequestJson(String jsonString, String[] ignoredFields)
             throws JsonProcessingException {
-        JsonNode unfilteredJsonNode = mapper.readTree(jsonString);
+        Map<String, Object> jsonMap = mapper.readValue(jsonString, Map.class);
+        // alternative way of removing fields we want to ignore when comparing
+        // 2 jsons & may be simpler than the logic that we have currently in place?
+        Map<String, Object> jsonMapFiltered = jsonMap.entrySet().stream()
+                .filter(e -> !Arrays.asList(ignoredFields).contains(e.getKey()))
```

Comment:
the idea would be that we can apply this is to any json and specify fields we'd want to keep, like the igo lims properties by basically just reversing the logic here

```
                .filter(e -> !Arrays.asList(ignoredFields).contains(e.getKey()))
```
switch to 
```
                .filter(e -> Arrays.asList(ignoredFields).contains(e.getKey()))
```